scarlet/observation.py

```diff
@@ -105,74 +124,64 @@ class Observation(Scene):
         prevent artifacts due to the FFT.
     """
 
-    def __init__(self, images, psfs=None, weights=None, wcs=None, filtercurve=None, padding=3, structure=None):
+    def __init__(self, images, psfs=None, weights=None, wcs=None, filtercurve=None, structure=None,
+                 padding=10):
         super().__init__(images.shape, wcs=wcs, psfs=psfs, filtercurve=filtercurve)
 
         self.images = np.array(images)
```

Comment:
why are images, weights, PSFs copied? They should never be altered, so changes should not be necessary.

Reply:
Let's make this a full code review: #101 

Not to be changed here

Reply:
Wrapping a sequence with `np.array` doesn't copy the object, it just creates a view to the contents of the sequence with the numpy array methods. So if someone passed a list to `images`, for example, then calling `B, Ny, Nx = images.shape` would raise an Exception if we didn't use the `np.array` wrapper. Internally any numpy function that you call (eg. `np.sum`) automatically does this at very little cost, so I don't think that it should be an issue. Numpy only copies if you use the `copy` method.